API_Guidelines.md

```diff
@@ -81,13 +89,20 @@ Payload
 :
 ```javascript
 {
-  "movies": {
+  "movie": {
     "id": 2,
     "title": "Goodfellas",
     "year": "1990"
   }
 }
 ```
+Payload (If no data is found, then `findRecord` returns an null value)
```

Comment:
this is more about APIs and not Ember Data, so `findRecord` should be `api.com/movies/20305` or some resource that doesn't exist. i'm not sure you would ever `get` a resource that exists and not get content back